src/Redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
```

Comment:
- [ ] In the `Categories` component, kindly create a function that dispatches this action creator and include JSX in the component that displays the **_under construction_** string when the `Check Status` button is clicked. 👍